docs/can-guides/commitment/recipes/file-navigator/file-navigator-advanced.md

```diff
@@ -115,7 +117,7 @@ Next, let’s make those entities, create a `store` to house them, and trap AJAX
 requests to use that `store`:
 
 @sourceref ./file-navigator-advanced-2.js
```

Comment:
If I understand this section correctly, the solution is broken up into two parts here. I think we should either:

1) Bring them together so the reader only updates the JS once.
2) Leave them separated, but only use one file and make sure the first part is highlighted.